src/node/wrapper.ts

```diff
@@ -314,7 +314,7 @@ export class ParentProcess extends Process {
         CODE_SERVER_PARENT_PID: process.pid.toString(),
         NODE_OPTIONS: `--max-old-space-size=2048 ${process.env.NODE_OPTIONS || ""}`,
       },
-      stdio: ["ipc"],
+      stdio: ["inherit", "pipe", "pipe", "ipc"],
```

Comment:
I'm thinking we could use `inherit` so we can get rid of the manual `pipe` calls below.

```suggestion
      stdio: ["inherit", "inherit", "inherit", "ipc"],
```

Reply:
Uh, so, this is embarrassing...I was looking at the wrong file. We could still use inherit but we'd need to remove the `pipe` calls to `process.stdout` and `process.stderr` and pipe them to the logger instead of the child. 

I say we just leave these at `pipe` for now haha. Sorry for the noise. :sob:

Reply:
LOL no worries at all! Glad you caught it before we merged. I'm less familiar with all this `pipe` stuff so kinda following your lead. Okay will do!